src/component/toolbox/feature/DataView.ts

```diff
@@ -371,6 +371,13 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
         addEventListener(closeButton, 'click', close);
 
         addEventListener(refreshButton, 'click', function () {
+            if ((typeof contentToOption === 'undefined' && typeof optionToContent !== 'undefined') ||
+                (typeof contentToOption !== 'undefined' && typeof optionToContent === 'undefined')) {
+                console.warn('It seems you have just provided one of `contentToOption` and `optionToContent` functions but missed the other one. Data change is ignored.')
+                close();
```

Comment:
Usually warn is only displayed in dev mode.

So a __DEV__ condition should be wrapped here.
```ts
if (__DEV__) {
  console.warn('....');
}
```